examples/azure-native-sample-project/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Native sample project
+
+This project creates a VM instance in Azure Cloud using the Pulumi Kotlin Azure native SDK.
+
+## How to run
+
+1. Install Pulumi (see: [docs](https://www.pulumi.com/docs/install/)).
```

Comment:
Maybe we should also copy & paste this step here?

`curl -fsSL https://get.pulumi.com | sh`